content/docs/0.8.x/integrations/action_provider/index.md

```diff
@@ -5,20 +5,32 @@ weight: 10
 keywords: [0.8.x-integration]
 ---
 
-An *action-provider* is an implementation of a [*Keptn-service*](../custom_integration/#keptn-service) with a dedicated purpose. This type of service is responsible for executing a remediation action and therefore might even use another tool.  
+An *action-provider* is an implementation of a [*Keptn-service*](../custom_integration/#keptn-service) with a dedicated purpose. This type of service is responsible for executing a remediation action and therefore might even use another tool. The [Keptn CloudEvents](#cloudevents) an action-provider has to subscribe to is:
 
-The [Keptn CloudEvents](#cloudevents) an action-provider has to subscribe to is:
+- `sh.keptn.event.action.triggered`
 
-- sh.keptn.event.action.triggered
+## Write your custom Action-provider
 
-## Implement custom Action-provider
+Like a Keptn-service, an action-provider has the following characteristics: 
+
+* has a **subscription** to an event (i.e., `sh.keptn.event.action.triggered`)
+* sends a **started event** to inform Keptn about receiving the event and acting on it
+* processes functionality and can therefore leverage additional tools, e.g., through their REST interface
+* sends a **finished event** to inform Keptn about its execution status and the result 
+
+### Subscription to Keptn event
+
+An *action-provider* starts working, when receiving a Keptn CloudEvent of type:
+
+-  [sh.keptn.event.action.triggered](https://github.com/keptn/spec/blob/0.1.6/cloudevents.md#action-triggered)
```

Comment:
we will need to update the link to the spec once we have released the spec containing the [Keptn 0.8 Events](https://github.com/keptn/spec/pull/45)

Reply:
True. Therefore we have the issue (https://github.com/keptn/keptn.github.io/issues/667) that addresses this when the spec is online.